### PR TITLE
Update Cray.md for polaris

### PR DESCRIPTION
**Pull request by roblatham00**
## Author Checklist
* [ ] **Provide Description** 
      Particularly focus on _why_, not _what_. Reference background, issues, test failures, xfail entries, etc.
* [ ] **Commits Follow Good Practice**
      Commits are self-contained and do not do two things at once. 
      Commit message is of the form: `module: short description` 
      Commit message explains what's in the commit.
* [ ] **Passes All Tests**
      Whitespace checker. Warnings test. Additional tests via comments.
* [ ] **Contribution Agreement**
      For non-Argonne authors, check [contribution agreement](http://www.mpich.org/documentation/contributor-docs/). 
      If necessary, request an explicit comment from your companies PR approval manager.

### CODE REVIEW

File: doc/wiki/source_code/Cray.md [hunk 1]
```diff
@@ -4,20 +4,34 @@ This page describes how to build mpich from the
...
+
+## Slingshot
+
+On Cray systems the vendor-supplied libfabric should be used to get slingshot support
+
+
+### Configure options
+
+You can leave off the 'cuda' options if you are e.g. testing only I/O or building.
+The Cray environemnt doesn't provide convienence environment variables for the 'pmi' or 'libfabric' paths, so you'll have to update these by hand when necessary
```
Comment (raffenet) [inline doc/wiki/source_code/Cray.md:22]: You can use `CRAY_PMI_PREFIX` in place of the hardcoded path. Nothing I can see for libfabric, though.

Comment (roblatham00) [reply doc/wiki/source_code/Cray.md]: thanks, pushed new version with that fix

**Comment by roblatham00** [comment]
test:mpich/whitespace

**Review by raffenet — APPROVED** [review]
LGTM

**Pull request merged**